scripts/validation/base-doc-combined.json

```diff
@@ -0,0 +1,711 @@
+            },
+            "userId": {
+              "type": "string",
+              "description": "The ID of the user"
+            },
+            "amount": {
+              "type": "number",
+              "description": "The total amount of the order"
+            }
+          }
+        }
+      }
+    },
+    "parameters": {
+      "userId": {
+        "description": "Id of the user."
+      }
+    },
+    "correlationIds": {
+      "default": {
+        "description": "Default Correlation ID",
+        "location": "$message.header#/correlationId"
+      }
+    },
+    "operations": {
+      "sendUserSignUp": {
+        "action": "send",
+        "title": "User sign up",
+        "channel": {
+          "$ref": "#/channels/user"
+        },
+        "bindings": {
+          "$ref": "#/components/operationBindings/sendUser"
+        },
+        "traits": [
+          {
+            "$ref": "#/components/operationTraits/binding"
+          }
+        ],
+        "reply": {
+          "$ref": "#/components/replies/signupReply"
+        }
+      }
+    },
+    "replies": {
+      "signupReply": {
+        "address": {
+          "$ref": "#/components/replyAddresses/signupReply"
+        },
+        "channel": {
+          "$ref": "#/channels/userSignupReply"
+        }
+      }
+    },
+    "replyAddresses": {
+      "signupReply": {
+        "location": "$message.header#/replyTo"
+      }
+    },
+    "securitySchemes": {
+      "userPasswordAuth": {
+        "type": "userPassword",
+        "description": "User password security scheme"
+      },
+      "apiKeyAuth": {
+        "type": "apiKey",
+        "description": "API key security scheme",
+        "in": "user"
+      },
+      "X509Auth": {
+        "type": "X509",
+        "description": "X509 security scheme"
+      },
+      "symmetricEncryptionAuth": {
+        "type": "symmetricEncryption",
+        "description": "Symmetric encryption security scheme"
+      },
+      "basicAuth": {
+        "type": "http",
+        "description": "Basic security scheme",
+        "scheme": "basic"
+      },
+      "httpApiAuth": {
+        "type": "httpApiKey",
+        "name":"API Key Sample",
+        "description": "api_key",
+        "in":"header"
+      },
+      "JWTAuth": {
+        "type": "http",
+        "scheme": "bearer",
+        "bearerFormat": "JWT"
+      },
+      "saslAuth": {
+        "type": "scramSha256",
+        "description": "SASL security scheme" 
+      },
+      "oauth": {
+        "type": "oauth2",
+        "description": "The oauth security descriptions",
+        "flows": {
+          "clientCredentials": {
+            "tokenUrl": "https://example.com/api/oauth/dialog",
+            "availableScopes": {
+              "subscribe:auth_revocations": "Scope required for authorization revocation topic"
+            }
+          }
+        },
+        "scopes": [
+          "subscribe:auth_revocations"
+        ]
+      }
+    },
```

Comment:
@smoya Please check this format of the document for the `securitySchemas` field if it's valid or not.

Reply:
LGTM!